### PR TITLE
#57 Created "chat" component

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-pulse/issues/57 |
| Decisions |  Created chat component that allows to send and receive messages. This chat component gives the ability for the users to share attachments. |
| Animated GIF | chat_sending_messages<br>**Responsive:** chat_responsive<br>**Can add attachments by using the attachments button or simply dragging & dropping the files:** chat_adding_attachments |

### CODE REVIEW

File: vue/components/ui/organisms/chat/chat.stories.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                 />
+                 <div>
+                    <p v-for="(message, index) in messagesData">
+                        {{ index }} <br>
+                        {{ message.avatarUrl }}
+                        {{ message.username }} {{ message.date }} <br>
+                        {{ message.messageContent.text }} <br>
+                        Attachments Length: {{ message.messageContent.attachments.length }}  Reactions Length: {{ message.messageContent.reactions.length }}
+                    </p>
+                 </div>
```
Comment (gcandal) [inline vue/components/ui/organisms/chat/chat.stories.js]: Delete the `div`
Change the `p` to `div`
Change every line to be a `p`

**Comment by joamag** [comment]
@gcandal 
1. We need to move the send button inside the chat area
2. We need to enable and disable the send button by having or not having content to be sent

**Comment by gcandal** [comment]
@NFSS10 try to find a fix for the attachment scroll that doesn't involve specifying `height`.

**Comment by joamag** [comment]
@NFSS10 🎆 🍾

**Pull request merged**